client/cmd/kcoin/main.go

```diff
@@ -331,3 +338,26 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 		go selfUpdater.Run()
 	}
 }
+
+func mustBeLatestMajorVersion(ctx *cli.Context) {
+	repository := ctx.GlobalString(utils.VersionRepository.Name)
+	finder := version.NewFinder(repository)
+	latest, err := finder.Latest(runtime.GOOS, runtime.GOARCH)
+	if err != nil {
+		log.Error("Error parsing current version, exiting checker", "err", err)
+		return
+	}
+
+	current, err := semver.Make(params.Version)
+	if err != nil {
+		log.Error("Error parsing current version, exiting checker", "err", err)
+		return
+	}
+
+	assetVersion := latest.Semver()
+	if assetVersion.Major >= current.Major {
+		log.Warn("Exiting client version is outdated", "current", current.String(), "latest", latest.Semver().String())
+		debug.Exit()
+		os.Exit(1)
```

Comment:
is this right

Reply:
Should it not be `if assetVersion.Major > current.Major`? Otherwise minor and patch versions will also trigger an exit, won't they?

Reply:
your right, had that for manual testing